core/chains/evm/client/errors.go

```diff
@@ -666,7 +676,9 @@ var drpc = ClientErrors{
 
 // Linkpool, Blockdaemon, and Chainstack all return "request timed out" if the log results are too large for them to process
 var defaultClient = ClientErrors{
-	TooManyResults: regexp.MustCompile(`request timed out`),
+	TooManyResults:     regexp.MustCompile(`request timed out`),
```

Comment:
Oh, sorry if what I wrote was unclear. I wasn't suggesting anything be moved from `arbitrum` to here--I think this belongs in the arbitrum list of client errors where you had it before.

All I was requesting was that `TooManyResults` be updated, because LogPoller looks for that error and needs to respond to it properly on arbitrum. On most rpc servers if there are too many results it will respond with the message "request timed out". But since it's formatted slightly differently on Arbitrum that should be part of the reg ex:
```suggestion
	ServiceUnavailable: regexp.MustCompile(`(: |^)503 Service Temporarily Unavailable(:|$)`),
```
```suggestion
	TooManyResults:     regexp.MustCompile(`request timed out|408 Request Timeout`),
```
We could separate it off into a separate list for arbitrum but I don't think that's necessary since the error message is nearly identical to the default message.